src/advanced/node-management.md

```diff
@@ -698,11 +698,11 @@ smallest and largest heights traversed to collect at most `count` blocks.
 The JSON object of the explored block range `range` and the array `blocks` of
 the `BlockHeader` objects. The range specifies the largest and the smallest
 heights of blocks that have been traversed to collect at most `count` blocks.
-The largest height `to` equals to `latest` if provided or to the height of
-the latest block in the blockchain, the smallest height `from` takes values
+The largest height `end` equals to `latest` if provided or to the height of
```

Comment:
Isn't `end` exclusive, as implied [by using `Range`](https://github.com/exonum/exonum/blob/ca48622516f821d7f5ffe5d907953a28d5e6e5e3/exonum/src/api/public/blockchain_explorer.rs#L32) in the implementation? If yes, it should be "The largest height `end` equals to `latest + 1`", right?